WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.kt

```diff
@@ -466,11 +524,83 @@ class ReaderPostDetailFragment : ViewPagerFragment(),
 
         viewModel.navigationEvents.observeEvent(viewLifecycleOwner, { it.handleNavigationEvent() })
 
-        val bundle = savedInstanceState ?: arguments
-        val isFeed = bundle?.getBoolean(ReaderConstants.ARG_IS_FEED) ?: false
-        val interceptedUri = bundle?.getString(ReaderConstants.ARG_INTERCEPTED_URI)
+        if (commentsSnippetFeatureConfig.isEnabled()) {
+            conversationViewModel.snackbarEvents.observe(viewLifecycleOwner, { event ->
+                if (!isAdded) return@observe
 
-        viewModel.start(isRelatedPost = isRelatedPost, isFeed = isFeed, interceptedUri = interceptedUri)
+                val fm: FragmentManager = childFragmentManager
+                val bottomSheet =
+                        fm.findFragmentByTag(NOTIFICATIONS_BOTTOM_SHEET_TAG) as? CommentNotificationsBottomSheetFragment
+                if (bottomSheet != null) return@observe
+                event.applyIfNotHandled {
```

Comment:
This is actually a (maybe not great :) ) way to avoid the post fragment consuming the snackbar event when the bottom sheet is opened, so that the bottom sheet can consume the snackbar event. It's how it worked already in the comments screen. We can in case review but I would maybe not block this PR, wdyt?

Reply:
Yeah, that's fine with me. Just wanted to confirm the logic :)